.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Description
+
+
+## Related Issue \#
+
+
+## Reviewer Testing Instructions
+
+
+## Submission Checklist
+I confirm that I have...
+- [ ] removed hard-coded Cognito IDs
```

Comment:
Is this an ongoing thing to include in a template?

Reply:
For the foreseeable future our demos and integration tests will require a hard-coded Cognito ID. As long as that's a requirement, there will be a risk of accidentally checking that into source control.

I've experimented with using git commit hooks to prevent code from being committed if it contains a Cognito ID. I have a working prototype, but I'm not sure if that's a solution we want to adopt or not. We should probably have some team discussion about that option.